ui/app/clinical/common/views/visitSummary.html

```diff
@@ -28,65 +25,79 @@
-                                              "true"></bahmni-diagnosis>
+            <div ng-switch-when="vitals" class="section">
+                <div ng-if="::section.config" id="vital-section">
+                    <section class="block observation dashboard-section" id="vitalSection">
+                        <ul bm-gallery access-impression="true">
+                            <bahmni-observation patient="::patient" visit-uuid="{{::visitUuid}}"
+                                                config="::section.config"
+                                                section-title="::section | titleTranslate"
+                                                class="section-title"></bahmni-observation>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+            <div ng-switch-when="consultationNotes" class="section">
+                <div ng-if="::section.config" id="consultationNotes-section">
+                    <section class="block observation dashboard-section" id="consultationNotesSection">
+                        <ul bm-gallery access-impression="true">
+                            <bahmni-observation patient="::patient" visit-uuid="{{::visitUuid}}"
+                                                config="::section.config"
+                                                section-title="::section | titleTranslate"
+                                                class="section-title"></bahmni-observation>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+            <div ng-switch-when="diagnosis" class="section">
+                <section ng-if="::section.config" class="block dashboard-section section-title has-link">
+                    <bahmni-diagnosis patient-uuid="::patient.uuid" config="::section" show-latest-diagnosis="false"
+                                      show-ruled-out-diagnoses="::section.showRuledOutDiagnoses"></bahmni-diagnosis>
+                    </bahmni-diagnosis>
                 </section>
             </div>
-            <div ng-switch-when="pivotTable">
+            <div ng-switch-when="pivotTable" class="section">
                 <div ng-if="::section.config">
                     <section pivot-table visit-uuid="::visitUuid" config="::section.config"
                              patient-uuid="::patient.uuid" display-name="::section | titleTranslate"></section>
                 </div>
             </div>
-            <div data-ng-switch-when="investigationResult">
+            <div data-ng-switch-when="labOrders" class="section">
                 <div ng-if="::section.config">
                     <investigation-results
                             params="::section.config"></investigation-results>
                 </div>
             </div>
-            <div data-ng-switch-when="radiology">
```

Comment:
any reason for removing radiology or pacsorder?

Reply:
Bahmni LITE doens't have pacs/radiology order configured. But in hindsight, realised we might need it. Can add it back when we add Radiology orders story. Just configuration in visit.json.